src/quo/components/wallet/token_input/view.cljs

```diff
@@ -20,17 +20,18 @@
    (string/upper-case (or (clj->js text) ""))])
 
 (defn input-section
-  [{:keys [token-symbol on-token-press value error? on-swap currency-symbol]}]
-  (let [theme        (quo.theme/use-theme)
-        window-width (:width (rn/get-window))]
+  [{:keys [token-symbol on-token-press value error? on-swap currency-symbol show-token-icon?
+           swappable?]}]
```

Comment:
Two new features of `token-input`:
If not `swappable?` we won't display the button to swap between crypto and fiat values.
If not `show-token-icon?` we do not show icon. Because `GWEI` or `UNITS` that we enter in transaction settings screens do not have icons.